libs/common/src/api/contract.ts

```diff
@@ -27,7 +27,7 @@ function ZodRegex(rx: RegExp, message: string) {
 const Location = z.array(z.number()).min(2).max(2);
 const DatasetId = z.string().regex(Rx.UrlSafeBase64);
 const DatasetItem = z.object({}).passthrough();
-const Dataset = z.array(Location);
+const Dataset = z.array(Location.nullable());
```

Comment:
sorry, pedantic request but think this is the best time to do it: please can you rename this type to DatasetLocations to be a bit clearer?

Reply:
Ok, 2 commits for this added to the end of the PR.

Reply:
Looks good